### DISCUSSION TITLE
August 2024 Conflux Documentation Contributor Incentives

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#731] [Announcements]
We are excited to announce the eligibility list of contributors who are qualified for contributor incentives for 202408. We deeply appreciate your support and highly value the insights and expertise you bring to our community.

以下为202408有资格获取激励的贡献者名单。非常感谢名单中的各位对本项目的支持。

The following GitHub contributors are eligible:

以下的GitHub贡献者有资格领取激励：

* [0xffffb](https://github.com/0xffffb)

* [jackleeio](https://github.com/jackleeio)

Additionally, the following GitHub contributors will receive previously undistributed incentives in this round：

此外，以下GitHub贡献者会在本期收到之前未下发的激励：

* [SPCFXDA](https://github.com/SPCFXDA)

* [wongzigii](https://github.com/wongzigii)

The eligibility list is curated in accordance with our [CONTRIBUTING guideline](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING). If you believe your contribution has been overlooked, please do not hesitate to contact us.

名单是根据[CONTRIBUTING指导原则](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING)所挑选的。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。

## Contributor Incentives Details 贡献者激励详情

We have introduced financial incentives for contributors which are directly related to the quality of the contributor's work.

我们为仓库的贡献者引入的经济激励是与贡献者的工作质量所直接相关的。

For incentives for documentation content contributors, please refer to:

https://www.notion.so/confluxglobal/3a41a670cac64d8b80a957f6318f6b06?v=791615efb1fb494eb74caf11d291d28a

and their incentives are calculated according to the [2024Q2 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md). If you have relevant contributions but have been omitted from the list, please feel free to contact us. In addition, we will collect the (temporary) addresses provided by the relevant contributors for airdrop redemption in this discussion.

文档内容贡献者的激励请参考：

https://confluxglobal.notion.site/3a41a670cac64d8b80a957f6318f6b06?v=791615efb1fb494eb74caf11d291d28a

其激励根据[2024Q2 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md)计算。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。此外我们将在本discussion中收集相关贡献者提供的领取空投的（临时）地址。

Thank you for your assistance in enhancing our project. If you have any queries or require assistance, please feel free to create an issue in our Github repository.

感谢各位持续帮助我们改进项目。如果你还有任何疑问或需要帮助，请随时在我们的Github仓库中创建issue。

**Comment by jackleeio** [comment]
https://github.com/Conflux-Chain/conflux-documentation/discussions/644#discussioncomment-10111369
This part has not been received yet, and the one for July has not been distributed either

**Reply by darwintree** [reply]
Sorry for the late response. Just got response from relating collegues that the rewards are already reviewed but waiting to be approved.

The last month's reward will be distributed together with this month's. We will urge the process to be done as soon as possible

**Reply by jackleeio** [reply]
🤣🤣🤣

**Reply by darwintree** [reply]
> 🤣🤣🤣

Would you check if the reward is distributed with expected amount?

**Reply by jackleeio** [reply]
Yes, thanks
